src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -34,6 +22,21 @@
                     Maintainer
                   %i.fas.fa-spinner.fa-spin.d-none
 
+          - component.with_header do
+            = link_to(user_path(user)) do
+              = display_name(user)
```

Comment:
We need to handle longer usernames to fix issues like in the screenshot below:

![long-strings](https://user-images.githubusercontent.com/1102934/218705777-b2618088-43ab-4dcb-bda9-df9d07c0962c.png)

Reply:
That's a good catch, but it should also be handled in the [`card-component`](https://github.com/openSUSE/open-build-service/pull/13809) implementation/improvement I'd say.
I'll take care of it.

Reply:
https://github.com/openSUSE/open-build-service/pull/13809/commits/78e11f25bb3e4f96221634aa04cfa0cf4c54a179